### DISCUSSION TITLE
Clue - closest teleport(s) or ability to add a note

### DISCUSSION THREAD

**Discussion by devil4072** [runelite/runelite#14999] [Q&A] [answered]
It would be awesome if for every clue step it said not only the location but the closest teleport.

Some do say the fairy ring code etc but other than that I don't recall many that say the closest teleport(s) (eg it could show the 3 closest teleports) - even if this was something that players could add themselves from within?

eg: O BIRDZ A ZANY EN PC [hard anagram] is cap'n izzy no-beard in brimhaven - and it could say closest teles: 1) spirit tree, 2) ardy & boat, 3) glory (to karamja).

I'd be happy to contribute to these if the option was added to be able to add tele details!

**Comment by Nightfirecat** [comment, marked as answer]
#13336 already exists for this request. We don't currently have a design in mind for this--user-configurable notes for clue scrolls would likely be what we'd want to add.
